docs/landing.md

```diff
@@ -0,0 +1,46 @@
+# Spark Integration Hub K8s documentation
+
+Spark Integration Hub K8s is an open-source [Kubernetes charm](https://juju.is/docs/olm/charmed-operator) that enables the integration of the Charmed Apache Spark solution with existing charms (PostgreSQL, S3-integrator, etc.) or bundles (COS, Identity, etc.) and properly configure the corresponding option in the Spark ecosystem.
+
+The Spark Integration Hub K8s charm provides centralization for other charmed components and generates the configuration options for the Spark jobs that will be run using Charmed Spark tooling (e.g. Spark-client snap, spark8t library, charmed applications).
+
+The needs met by the Spark Integration Hub K8s charm are the following:
+
+- Handle relations with other charms and translate them into configuration options that Spark can handle.
+- Store the configuration options in Kubernetes secrets for the service account managed by the Apache Charmed Spark solution (all service accounts are labeled with predefined labels)
+- Handle the addition/removal of service accounts with corresponding updates of configurations.
+- Handle updates from the related charms (i.e., new database credentials, change of endpoints, updated Azure storage credentials.)
+- The configuration hub charm may also be able to create and configure services accounts that can run Spark jobs.
```

Comment:
I think this text covers what the product does, rather than what needs it meets.

```suggestion
```

Suggestion:
I'd suggest removing it from here (concentrating on the four main questions) and potentially adding it in the middle section (before the Project and Community). Maybe even with some **instructions on how to deploy and use the charm**.

Reply:
Removed this altogether. This will be more impactful as how-to entries in the navigation table